lib/mincer/manifest.js

```diff
@@ -176,6 +176,7 @@ function gzip(data) {
  *  - `embedMappingComments` (false) - set `true` to embed sourcemap url
  *    into created files
  *  - `compress` (false) - set `true` to also create gzipped files
+ *  - `preventXSSI` (true) - set `false` to skip adding XSSI protection header
```

Comment:
@puzrin thanks, I changed the option to `noSourceMapProtection` as you suggested. Can you please check if it's OK?